tests/ui/test_ui_tools.py

```diff
@@ -1247,7 +1247,9 @@ def mock_external_classes(self, mocker):
                             return_value=(64, 64))
         mocker.patch(VIEWS + '.urwid.SimpleFocusListWalker', return_value=[])
         self.about_view = AboutView(self.controller, 'About',
-                                    zt_version='0.5.1+git')
+                                    zt_version='0.5.1+git',
+                                    server_version='2.2-dev',
+                                    server_feature_level='1')
```

Comment:
I have used hard-coded values here as `self.about_view` is used in the other tests that don't 'care' about the version or the feature_level (see also the other in-line comment in this commit).